src/dash-adapter.js

```diff
@@ -186,6 +187,9 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
     if (Utils.Object.hasPropertyPath(config, 'text.useNativeTextTrack')) {
       adapterConfig.textTrackVisibile = Utils.Object.getPropertyPath(config, 'text.useNativeTextTrack');
     }
+    if (Utils.Object.hasPropertyPath(config, 'text.useShakaTextTrackDisplay')) {
+      adapterConfig.useShakaTextTrackDisplay = Utils.Object.getPropertyPath(config, 'text.useShakaTextTrackDisplay');
```

Comment:
let's use the native shaka config object for now - that comes from playback.options.html5.dash

Reply:
It's a bit not what playback.options.html5.dash means, it means to pass shaka config.
also, we use this config on playkit as well and use config.playback.options.html5.dash on playkit will be super weird

Reply:
as discussed - if we reuse `useNativeTextTrack` we won't need another flag there

Reply:
but it's meaningless to use this flag